# Set LED state (0-18)
uint8 state
---
bool success
string message
